doc/conf.py

```diff
@@ -137,16 +137,17 @@
 copyright = f"2017-{year}, The PyGMT Developers"  # pylint: disable=redefined-builtin
 if len(__version__.split("+")) > 1 or __version__ == "unknown":
     version = "dev"
+    html_baseurl = "https://pygmt.org/dev/"
 else:
     version = __version__
+    html_baseurl = "https://pygmt.org/latest/"
```

Comment:
```suggestion
if len(__version__.split("+")) > 1 or __version__ == "unknown":
    # Set base_url for stable version
    version = "dev"
    html_baseurl = "https://pygmt.org/dev/"
else:
    version = __version__
    # Set base_url for dev version
    html_baseurl = "https://pygmt.org/latest/"
```

I think there should be comments explaining this `if` statement. My wording is  rough suggestion, but I think it makes it easier for future readers to understand why we have two different options for `html_baseurl`.